common-content/en/blocks/morning-orientation/index.md

```diff
@@ -0,0 +1,41 @@
++++
+title="🎡 Morning orientation"
+headless="true"
+time=15
+[objectives]
+    1="Introduce and welcome new volunteers"
+    2="Nominate a timekeeper"
+    3="Check the morning day plan and group structures"
++++
+
+{{<note title="Planning during the week">}}
+🧭 Step 1 and step 6 can be done earlier in the week. Create a post on Slack and get some people to take on the roles of **timekeeper** and **facilitator**. Nominate people who've not done it before.
+{{<n/ote>}}
+
+# 👣 Steps
+
+0. Assemble the entire group (all volunteers & all trainees) in a circle
+1. Choose someone (volunteer or trainee) to be a **facilitator** for this morning orientation block
+2. The **facilitator** should briefly welcome everyone with an announcement. Here is an example announcement to read aloud:
+
+> 💬 "Morning everyone, Welcome to CYF {REGION}, this week we are working on {MODULE HERE} {SPRINT NUMBER HERE} and we're currently working on {SUMMARISE THE TOPICS OF THE WEEK}"
+
+3. Check if there are any new volunteers or if there are some attendees who haven't met before. Ask any newcomers to introduce themselves to the group/relevant people
```

Comment:
Is this not really an energiser? Can this come before energiser or replace it in some circumstances?

Reply:
I think the energiser is a separate thing. But I see no reason why it can't be replaced by an orientation session by those running the session. Yeah I think it can also come first too.